server/pulp/server/db/manage.py

```diff
@@ -180,6 +185,12 @@ def ensure_database_indexes():
         model_class.ensure_indexes()
 
 
+def _get_running_worker_names():
```

Comment:
Since the function provides filtering at the collection level, putting them on the QuerySet associated with the Worker is most consistent with the mongoengine (and future Django) design. In fact, the `status.get_workers()` method should not even exist, but since each PR should handle one thing leaving that there for now I think is OK.